src/applications/personalization/profile360/actions/index.js

```diff
@@ -8,7 +8,7 @@ export const FETCH_MILITARY_INFORMATION_SUCCESS =
 export const FETCH_ADDRESS_CONSTANTS_SUCCESS =
   'FETCH_ADDRESS_CONSTANTS_SUCCESS';
 
-async function getData(apiRoute) {
+export async function getData(apiRoute) {
```

Comment:
It doesn't feel like this should be exported from an actions file. It's a helper function, if we need it in multiple places, maybe it makes sense in the api utilities?

Reply:
Moved to utils